docs/scipp_widgets_intro.ipynb

```diff
@@ -59,18 +55,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from scipp_widgets.input_spec import InputSpec, StringInputSpec\n",
+    "from scipp_widgets.input_spec import InputSpec, TextInputSpec\n",
     "data_input = InputSpec('x', tooltip='input data')\n",
-    "dimension_input = StringInputSpec('dim', options = ('x', 'z'))"
+    "dimension_input = TextInputSpec('dim', options = ('x', 'z'))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `eval_input` allows variables in the notebooks global scope to be passed to this input by name.\n",
-    "\n",
-    "A widget which wraps the plot function can then be constructed like so:"
+    "A simeple user interface which wraps the sum function can then be constructed like so:"
    ]
   },
   {
```

Comment:
I can't comment on the line i want because you made no changes there, but I think the `DisplayWidget` is so standard that specifying the `sc.plot.plot` function and the `data_input` creation should just be hidden from the user.
`DisplayWidget` would simply give you the finished widget.

You could then make a more advanced version, which would allow more options, such as changing a colormap or applying a log scale, but since we have now made a lot of that functionality available with buttons around the plot, I am guessing just a simple widget that calls plot would cover most cases.

Also, since there is only one `plot` function, you don't really need to specify the `plot` function every time you create such a widget.